sunpy/util/types.py

```diff
@@ -0,0 +1,9 @@
+"""
+A collection of ABCs to use to shortcut having to import things.
+"""
+import abc
+
```

Comment:
```suggestion


```

Reply:
You don't need to import abc for metaclasses to work?

Reply:
The suggestion is adding an extra line for the pep8 checker.